Dockerfile

```diff
@@ -1,19 +1,26 @@
 FROM ubuntu:14.04
 
-RUN apt-get update && \
-    apt-get -y install software-properties-common wget && \
-    apt-add-repository 'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main' && \
-    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
-    apt-get update && \
-    apt-get -y install python python-pip libpq-dev python-dev postgresql-client-9.4 && \
-    apt-get clean
+RUN apt-get update
+RUN apt-get -y install software-properties-common wget curl
+RUN apt-add-repository 'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main'
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+RUN apt-get update
+RUN apt-get -y install python python-pip libpq-dev python-dev postgresql-client-9.4
+RUN curl -sL https://deb.nodesource.com/setup_4.x | bash -
+RUN apt-get install -y nodejs
+RUN apt-get clean
```

Comment:
The reason it was one `RUN` statement is to reduce the number of layers customer machines need to pull.  I'd like to keep it with only 2 layers (the way it was): 1 layer that sets up the base packages and such, and one that copies and builds _our_ stuff.

If it's useful to you, you can of course split out the layers during your own development.  Caching is useful there.